pkg/cluster/info.go

```diff
@@ -40,12 +39,13 @@ type Info struct {
 	Name             string
 	RestConfig       *rest.Config
 	ClientProducer   client.Producer
+	Broker           *v1alpha1.Broker
 	Submariner       *v1alpha1.Submariner
 	ServiceDiscovery *v1alpha1.ServiceDiscovery
 	nodeCount        int
 }
 
-func NewInfo(clusterName string, config *rest.Config) (*Info, error) {
+func NewInfo(clusterName, brokerNamespace string, config *rest.Config) (*Info, error) {
```

Comment:
The `brokerNamespace` parameter is always `""`, let’s drop it for now and rely on the constant.

Reply:
For, `recover-brokerinfo` the namespace might be different and hence have added this parameter here.

Reply:
In this PR, it can’t be; you always pass in `""`.